src/__experimental__/components/radio-button/radio-button-group.component.js

```diff
@@ -12,14 +15,54 @@ function checkedTabIndex(checked) {
   return checked ? 0 : -1;
 }
 
+const getValidationType = ({ hasError, hasWarning, hasInfo }) => {
+  let type = 'help';
+
+  if (hasError) {
+    type = 'error';
+  } else if (hasWarning) {
+    type = 'warning';
+  } else if (hasInfo) {
+    type = 'info';
+  }
+
+  return type;
+};
+
+const getValidationMessage = (type, props) => {
+  const {
+    labelHelp,
+    errorMessage,
+    warningMessage,
+    infoMessage
+  } = props;
+
+  switch (type) {
+    case 'error':
+      return errorMessage;
+    case 'warning':
+      return warningMessage;
+    case 'info':
+      return infoMessage;
+    default:
+      return labelHelp;
+  }
+};
```

Comment:
we won't need these when you'll reuse the `ValidationIcon` component

Reply:
Same again, this has been resolved in FE-1728 PR.